Resources/config/image_variations.yml

```diff
@@ -9,3 +9,7 @@ system:
                 reference: reference
                 filters:
                     - {name: geometry/scalewidthdownonly, params: [300]}
+            platformui_profileview:
+                reference: reference
+                filters:
+                    - {name: geometry/scalewidthdownonly, params: [40]}
```

Comment:
you are displaying the avatar as a square, so the variation should generate a square. I think we should get that with something like (not tested):
```yml
- {name: geometry/scaledownonly, params: [40, 40]}
- {name: geometry/crop, params: [40, 40, 0, 0]}
```